op-chain-ops/cmd/celo-migrate/main.go

```diff
@@ -68,6 +68,14 @@ var (
 		Name:  "dry-run",
 		Usage: "Dry run the upgrade by not committing the database",
 	}
+	gasLimitFlag = &cli.Uint64Flag{
```

Comment:
Do we need to make this configurable? My understanding was that this is just for testing

Reply:
This allows you to migrate at pre gingerbread blocks, the gas limit changed a few times pre-gingerbread so I figured that people may want to set a variety of gas limits, depending on what block the transition occurred at.

In the real migration cases we will be migrating post gingerbread so people won't need to set the flag.

Reply:
I just wonder if setting this is important for testing? Does it matter if you can set the gas limit to 20M or if it is hardcoded to 10M? As you write in the real migration this won't be used.

Reply:
It's hard to say, might depend on the kind of testing you need to do. But I can just revert to a hardcoded value it works for my current use case.

What about the base fee flag, I guess you feel the same?